README.rst

```diff
@@ -49,7 +57,8 @@ Setup
 .. code:: json
 
     {
-        "token": "xxxxxxxx-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx-xxxxxxxxxxxxxxxxxxxxx"
+        "token": "xxxxxxxx-xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx-xxxxxxxxxxxxxxxxxxxxx",
+        "username": "yyyyyyyy-yyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyyy-yyyyyyyyyyyyyyyyyyyyy"
```

Comment:
I think it might be better to leave the examples as-is and add a separate section about multi-account support.